drivers/project_scorecard/app/models/project_scorecard/document_exports/scorecard_export.rb

```diff
@@ -41,7 +41,7 @@ def perform
         )
         PdfGenerator.new.perform(
           html: html,
-          file_name: "#{@report.project.name.titlecase} Scorecard #{DateTime.current.to_s(:db)}",
+          file_name: "#{@report.project.ProjectName.titlecase} Scorecard #{DateTime.current.to_s(:db)}",
```

Comment:
Why go from name to ProjectName here, but in `drivers/project_scorecard/app/views/project_scorecard/warehouse_reports/scorecards/_organization.haml` go from ProjectName to name?

Reply:
No good reason. This report will only be visible for confidential projects _if_ the user has permission to see confidential projects. So `project.name` and `project.ProjectName` are effectively the same. I can update it to `name(current_user)` in case we change that down the line?

Reply:
can't test this one because PDF generation is hanging on my branch - note to self to come back to it

Reply:
okay, fixed https://github.com/greenriver/hmis-warehouse/pull/2225/commits/b28d0b33e0aa71339869fd8f9ca7a776a4632d99